readme.md

```diff
@@ -26,7 +38,7 @@ Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
 ```text
-
+When the form is submitted you send an AJAX request with the type: 'POST' containing the data from the form to the matching backend route that can receive that data and do something with it.
 ```
```

Comment:
:+1:

Also looking for the steps required to trigger the event (event listener w/ `preventDefault()`) and pull information from the form (e.g., `$( "input" ).val();`).